src/widgets/IndustryBlock/IndustryBlock.tsx

```diff
@@ -18,22 +18,24 @@ export const IndustryBlock = () => {
       <SectionTitle text="Industry solutions" />
       <div
         className="flex flex-col items-start
-        mt-[20px]
-        sm:mt-[25px]
-        md:mt-[38px]
-        xl:mt-[60px]">
+          mt-[20px]
```

Comment:
Why are the indents different everywhere? Is there any way to make them more readable?